From 09a4eeb81e84908a10a5b9028e0e58fda129942a Mon Sep 17 00:00:00 2001
From: "Arnaud Patard \\(Rtp\\)" <arnaud.patard@rtp-net.org>
Date: Thu, 20 Jun 2013 23:07:06 +0200
Subject: i2c: imx: allow autoloading on dt ids

Allow udev to autoload the module when booting with device-tree

Signed-off-by: Arnaud Patard <arnaud.patard@rtp-net.org>
Signed-off-by: Wolfram Sang <wsa@the-dreams.de>
(cherry picked from commit 2f641a8bdb1b808b9bf1d0ca7d169d199aaf6ff4)
Signed-off-by: Darren Hart <dvhart@linux.intel.com>
---
 drivers/i2c/busses/i2c-imx.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/i2c/busses/i2c-imx.c b/drivers/i2c/busses/i2c-imx.c
index 6406aa960f2a..e24279725d36 100644
--- a/drivers/i2c/busses/i2c-imx.c
+++ b/drivers/i2c/busses/i2c-imx.c
@@ -147,6 +147,7 @@ static const struct of_device_id i2c_imx_dt_ids[] = {
 	{ .compatible = "fsl,imx21-i2c", .data = &imx_i2c_devtype[IMX21_I2C], },
 	{ /* sentinel */ }
 };
+MODULE_DEVICE_TABLE(of, i2c_imx_dt_ids);
 
 static inline int is_imx1_i2c(struct imx_i2c_struct *i2c_imx)
 {
-- 
1.8.5.rc3

